### PR TITLE
Reviewed and updated 19 files of the public API

### PR DESCRIPTION
**Pull request by HalynaDumych**
Modified the existing comments and added the missing comments to
the public APIs of the `olp-sdk-dataservice-read/lib/client/` lib.

Relates-to: OLPEDGE-869

Signed-off-by: Halyna Dumych <ext-halyna.dumych@here.com>

### CODE REVIEW

File: @here/olp-sdk-dataservice-read/lib/client/DataRequest.ts [hunk 1]
```diff
@@ -29,73 +29,106 @@ export class DataRequest {
     private version?: number;
     private billingTag?: string;
 
+    /**
+     * Gets a data handle for the request.
+     * 
+     * The data handle identifies a specific blob so that you can get the contents of that blob.
+     * The data handle can only contain alphanumeric characters, '-', and '.' [0-9, a-z, A-Z, -, .].
+     * The maximum length of this field is 1024 characters.
```
Comment (OleksiiZubko) [inline @here/olp-sdk-dataservice-read/lib/client/DataRequest.ts]: Please, remove it. We don't have any validations for now

File: @here/olp-sdk-dataservice-read/lib/client/CatalogsRequest.ts [hunk 2]
```diff
@@ -20,38 +20,50 @@
...
         return this.schemaHrn;
     }
 
     /**
-     * Set value of layer schema HRN to use in method getCatalogs from ConfigClient.
-     * If schema is setted, then getCatalogs will return filtered response by the specified layer schema HRN.
-     * If schema is not setted, then filter will return search for all.
-     * @param schemaHrn layer schema HRN
+     * Sets a value of a layer schema HERE Resource Name (HRN) that is used in the `getCatalogs` method of the [[ConfigClient]] instance.
+     * If the schema is set, the `getCatalogs` method returns a response filtered by the specified layer schema HRN.
```
Comment (OleksiiZubko) [inline @here/olp-sdk-dataservice-read/lib/client/CatalogsRequest.ts]: returns only catalogs witch have layer/layers with the specific schema

Comment (HalynaDumych) [reply @here/olp-sdk-dataservice-read/lib/client/CatalogsRequest.ts]: Fixed

File: @here/olp-sdk-dataservice-read/lib/client/DataRequest.ts (hunk 1 above)
Comment (HalynaDumych) [reply @here/olp-sdk-dataservice-read/lib/client/DataRequest.ts]: Fixed

**Pull request merged**